rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -206,18 +206,56 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 	m_emu_settings->EnhanceCheckBox(ui->spuLoopDetection, emu_settings_type::SPULoopDetection);
 	SubscribeTooltip(ui->spuLoopDetection, tooltips.settings.spu_loop_detection);
 
-	m_emu_settings->EnhanceCheckBox(ui->accurateXFloat, emu_settings_type::AccurateXFloat);
-	SubscribeTooltip(ui->accurateXFloat, tooltips.settings.accurate_xfloat);
+	// m_emu_settings->EnhanceCheckBox(ui->accurateXFloat, emu_settings_type::AccurateXFloat);
+	// SubscribeTooltip(ui->accurateXFloat, tooltips.settings.accurate_xfloat);
 
-	m_emu_settings->EnhanceCheckBox(ui->approximateXFloat, emu_settings_type::ApproximateXFloat);
-	SubscribeTooltip(ui->approximateXFloat, tooltips.settings.approximate_xfloat);
+	// m_emu_settings->EnhanceCheckBox(ui->approximateXFloat, emu_settings_type::ApproximateXFloat);
+	// SubscribeTooltip(ui->approximateXFloat, tooltips.settings.approximate_xfloat);
 
 	m_emu_settings->EnhanceCheckBox(ui->fullWidthAVX512, emu_settings_type::FullWidthAVX512);
 	SubscribeTooltip(ui->fullWidthAVX512, tooltips.settings.full_width_avx512);
 	ui->fullWidthAVX512->setEnabled(utils::has_avx512());
 
 	// Comboboxes
 
+	ui->xfloatAccuracy->addItem(tr("Accurate XFloat"));
+	ui->xfloatAccuracy->addItem(tr("Approximate XFloat"));
+	ui->xfloatAccuracy->addItem(tr("Relaxed XFloat"));
+	if (m_emu_settings->GetSetting(emu_settings_type::AccurateXFloat) == "true")
+		ui->xfloatAccuracy->setCurrentIndex(0);
+	else if (m_emu_settings->GetSetting(emu_settings_type::ApproximateXFloat) == "true")
+		ui->xfloatAccuracy->setCurrentIndex(1);
+	else
+		ui->xfloatAccuracy->setCurrentIndex(2);
+
+	connect(ui->xfloatAccuracy, QOverload<int>::of(&QComboBox::currentIndexChanged), ui->xfloatAccuracy, [this](int index)
+	{
+		if (index < 0) return;
+
+		if (ui->xfloatAccuracy->currentIndex() == 0)
+		{
+			m_emu_settings->SetSetting(emu_settings_type::AccurateXFloat, "true");
+			m_emu_settings->SetSetting(emu_settings_type::ApproximateXFloat, "false");
```

Comment:
why is there no setter for Relaxed?

Reply:
There is no point in disabling it so it remains a hidden setting